src/yardstick/cli/label.py

```diff
@@ -86,10 +95,9 @@ def compare_results_against_labels(
 @click.option("--summarize", "-s", is_flag=True, help="summarize each entry")
 @click.pass_obj
 def list_labels(_: config.Application, image: str, summarize: bool):
-    if image:
-        display_label_entries = store.labels.load_for_image(image)
-    else:
-        display_label_entries = store.labels.load_all()
+    display_label_entries = (
```

Comment:
Do you think it would be worth discussing whether we want these? The first one seems pretty easy to read to me, but I'm not up on what readers of Python will expect.

Reply:
I'm not against flipping this off if that's what folks want (I want to say it's a ruff rule that adjusted this) but have a soft preference to leaving it on.